jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -209,6 +247,14 @@ public Instant getCreationTime() {
     return labels;
   }
 
+  public boolean isEntrypointInferredFromBaseImage() {
```

Comment:
Wording nit: Since this is the configuration, the flag is more of like "should entrypoint be inferred from base image?" rather than "is entrypoint inferred from base image?". Perhaps this could be named `shouldInheritEntrypoint` (base image is implied)?

Reply:
Yes it's better.